app/views/content/how-to-apply-for-teacher-training/ske-explained-transcript.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Subject knowledge enhancement explained video transcript
+description: Full transcript for the 'Subject knowledge enhancement' video.
+---
+
+**Speaker 1:** SKE stands for subject knowledge enhancement and it allows you to build upon your subject knowledge that you would be expected to have as a teacher. And people do it if they maybe have an A level in the area but not a degree. 
```

Comment:
Should the `Speaker 1` sit on a separate line to the comments? Feels a bit squished at the moment?